### PR TITLE
Fix pylint warnings in middleman tests

### PR DESCRIPTION
**Pull request by Jakub89**

### CODE REVIEW

File: concent_api/middleman/tests/test_middleman.py [hunk 1]
```diff
@@ -77,45 +77,48 @@ def test_that_middleman_is_created_with_default_params(self):  # pylint: disable
 class TestMiddleManServer:
     patcher = None
     crash_logger_mock = None
+    internal_port = None
+    external_port = None
+    middleman = None
 
     @pytest.fixture(autouse=True)
-    def middleman_setup(self, unused_tcp_port_factory, event_loop):
+    def _middleman_setup(self, unused_tcp_port_factory, event_loop):
```
Comment (dybi) [inline concent_api/middleman/tests/test_middleman.py]: Please rename it to: `setup_middleman`

File: concent_api/middleman/tests/test_middleman.py [hunk 2]
```diff
@@ -77,45 +77,48 @@ def test_that_middleman_is_created_with_default_params(self):  # pylint: disable
...
         self.patcher = mock.patch("middleman.middleman_server.crash_logger")
         self.crash_logger_mock = self.patcher.start()
         self.internal_port, self.external_port = unused_tcp_port_factory(), unused_tcp_port_factory()
         self.middleman = MiddleMan(internal_port=self.internal_port, external_port=self.external_port, loop=event_loop)
         yield self.internal_port, self.external_port
         self.patcher.stop()
 
-    def test_that_if_keyboard_interrupt_is_raised_application_will_exit_without_errors(self, middleman_setup):
+    def test_that_if_keyboard_interrupt_is_raised_application_will_exit_without_errors(self, _middleman_setup):
```
Comment (dybi) [inline concent_api/middleman/tests/test_middleman.py]: remove `_middleman_setup` from here and the rest of test methods

**Comment by dybi** [comment]
One more thing, it seems that `timeout = 2` and `short_delay = 0.5` is a bit of overkill. Could you please tune these values a little bit? (I know I set them initially - mea coulpa ;))

File: concent_api/middleman/tests/test_middleman.py (hunk 1 above)
Comment (Jakub89) [reply concent_api/middleman/tests/test_middleman.py]: Done

File: concent_api/middleman/tests/test_middleman.py (hunk 2 above)
Comment (Jakub89) [reply concent_api/middleman/tests/test_middleman.py]: Done

**Pull request merged**